src/main/java/apoc/result/AssertSchemaResult.java

```diff
@@ -1,13 +1,29 @@
 package apoc.result;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class AssertSchemaResult {
     public final String label;
     public final String key;
+    public final List<String> keys;
     public boolean unique = false;
     public String action = "KEPT";
 
-    public AssertSchemaResult(String label, String key) {
+    public AssertSchemaResult(String label, List<String> keys) {
+        this.label = label;
+        if(keys.size() > 1) {
+            this.keys = keys;
+            this.key = null;
+        } else {
+            this.key = keys.get(0);
+            this.keys = null;
```

Comment:
single element list

Reply:
To do that would change the `yield` which people might be relying on - and to yield multiple keys in a field called 'key' would not give any indication of the actual content

Reply:
why? just put a single element list in keys for the single key case.
So that new users can always use `keys` if they want to

Reply:
Ahh so you didn't mean remove the `key` property - gotcha, makes sense